main.go

```diff
@@ -259,12 +260,24 @@ func main() {
 		fatal.Exit("gehen.yml must contain at least one service")
 	}
 
-	// Connect to ECS API
 	sess := session.Must(session.NewSession(&aws.Config{
 		Region: aws.String("us-east-1"),
```

Comment:
@cszatmary Its a bit sus that we are hardcoding `us-east-1` - If I take this out does it just infer region based on the aws config in shipit or an env var?

Reply:
yes this takes highest priority, if you skip it, it comes from the credential chain...